### ISSUE TITLE
PFG fails to generate formulas without carbon atoms

### ISSUE THREAD

**Issue by tomas-pluskal** [zmzhang/PFG#2]
Hi,

I've been testing your PFG generator. I noticed it fails to generate any formulas for this input:

```
PFG test -m 1000 -t 5  --H 0-100 --N 0-100 --O 0-100 --S 0-100 --P 0-100 -r
```

Is that a bug?

**Comment by tomas-pluskal** [comment]
Note that e.g. H38N23O22S9 has a mass of 1000.004830 (within 5 ppm tol)

**Comment by zmzhang** [comment]
Hi Tomas:

It is a bug, and we have fixed it already in [commit 0d6a481](https://github.com/zmzhang/PFG/commit/0d6a4813d8a5adc095d58388e72e16edf96236f9).
If there are no rules for filtering the formula, the return value of frules should be true.
The formula can be generated correctly, please see the following figure.
![frules](https://cloud.githubusercontent.com/assets/6937141/17427767/b219ce8a-5b15-11e6-8969-4eceec288eea.png)

Cheers!
Zhimin
